packages/protvista-structure/README.md

```diff
@@ -1,13 +1,41 @@
 # `<protvista-structure>`
+
 In progress - A native web component wrapper around LiteMol
 
 ## Usage
+
 ```html
-      <protvista-structure id="1AAP"></protvista-structure>
+<!-- Basic usage -->
+<protvista-structure id="1AAP" />
+<!-- With UniProt Accession -->
+<protvista-structure accession="P06493" />
+<!-- To highlight a position/range -->
+<protvista-structure accession="P06493" highlight="209:220" />
+<!-- With a specific PDBe entry selected and the entrie table being hidden -->
+<protvista-structure
+  accession="P06493"
+  highlight="209:220"
+  molecule="4YC3"
+  hide-table
+/>
 ```
 
 ## API Reference
 
 ### Properties
+
 #### `accession: UniProt accession`
-A protein accession
\ No newline at end of file
+
+A protein accession
+
+#### `highlightresidues`
```

Comment:
only `highlight` is valid, not `highlightresidues` (which needs to be removed). 
The description would be something like "highlight residues corresponding to the given sequence positions".